oxidation/crates/parsec_api_types/src/certif.rs

```diff
@@ -180,25 +181,7 @@ impl_verify_and_load_allow_root!(UserCertificate);
 impl_unsecure_load!(UserCertificate);
 impl_dump_and_sign!(UserCertificate);
 
-new_data_struct_type!(
-    UserCertificateData,
-    type: "user_certificate",
-
-    author: CertificateSignerOwned,
-    timestamp: DateTime,
-
-    user_id: UserID,
-    // Added in Parsec v1.13
-    #[serde(default, deserialize_with = "maybe_field::deserialize_some")]
-    human_handle: Option<Option<HumanHandle>>,
-    public_key: PublicKey,
-    // `profile` replaces `is_admin` field (which is still required for
-    // backward compatibility)
-    is_admin: bool,
-    // Added in Parsec v1.14
-    #[serde(default, deserialize_with = "maybe_field::deserialize_some")]
-    profile: Option<UserProfile>,
-);
+parsec_data!("schema/certif/user_certificate.json");
```

Comment:
How do we know which struct this macro creates? Maybe we should add a comment, or maybe this should be part of the macro (e.g the macro checks that the json files as generated a provided list of names).

@touilleMan what do you think?